e2e/Dockerfile

```diff
@@ -1,9 +1,9 @@
 # base image with Node.js and playwright preinstalled
-FROM mcr.microsoft.com/playwright:v1.48.1-noble as base
+FROM mcr.microsoft.com/playwright:v1.50.0-noble as base
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
 ENV NODE_ENV=test
-RUN apt-get -y update && apt-get install -y netcat-openbsd
+RUN apt-get -y update && apt-get install -y netcat-openbsd make g++
```

Comment:
I ran this through [hadolint](https://github.com/hadolint/hadolint) and it said...

```
Dockerfile:6 DL3008 warning: Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
Dockerfile:6 DL3015 info: Avoid additional packages by specifying `--no-install-recommends`
Dockerfile:6 DL3009 info: Delete the apt-get lists after installing something
```

Reply:
Thanks, I added a draft issue to track.